static/.htaccess

```diff
@@ -223,15 +213,13 @@ RewriteRule ^docs/8.6/apis-tools/community-clients/go-client/(.*)$ https://githu
 RewriteRule ^meta/ / [R=301,L]
 
 # Move SAP docs into Camunda integrations
-RewriteRule ^docs/components/early-access/sap/?$ /docs/components/camunda-integrations/sap/$1 [R=301,L]
+RewriteRule ^docs/components/early-access/sap/?$ /docs/components/camunda-integrations/sap/ [R=301,L]
 
 # Reroute element templates and HTTL to C7 docs
-RewriteRule ^docs/next/components/modeler/desktop-modeler/element-templates/c7-defining-templates/(.*)$ https://docs.camunda.org/manual/latest/modeler/element-templates/ [R=301,L]
```

Comment:
We're now in the 8.7 "vCurrent" section of the redirect rules.

At the time of release, I remove any of these `/next` redirects in the section corresponding to the freshly released version. In this case, it looks like I missed one. 

(The same applies to the next diff, too.)